### DISCUSSION TITLE
So long and thanks for all the fish! 🐟

### DISCUSSION THREAD

**Discussion by kasperisager** [Siteimprove/alfa#937] [General]
Today was my last day at @Siteimprove and so moving forward @Jym77 will be running the show. I of course hope the discussions section will be kept around and might pop in for a conversation every now and then.

Best of luck with the project and the community that is ever so slowly forming around it! 🚀

**Comment by lovebes** [comment]
thanks so much @kasperisager for all the help and best of luck in your next step in your career!!
